docs/core/event_handler/api_gateway.md

```diff
@@ -458,6 +458,12 @@ In the following example, we use a new `Header` OpenAPI type to add [one out of
 
     1. `cloudfront_viewer_country` is a list that must contain values from the `CountriesAllowed` enumeration.
 
+#### Serializing objects
+
+We support the serialization of various Python objects, including Pydantic models, dataclasses, enumerations, file paths, scalar types (strings, integers, floats, and None), dictionaries, various sequence types (lists, sets, frozen sets, generators, tuples, and deques), and others defined [here](https://github.com/aws-powertools/powertools-lambda-python/blob/develop/aws_lambda_powertools/event_handler/openapi/encoders.py#L24).
```

Comment:
I think I'd rather have this as an exhaustive list, it'll improve readability and stop people from having to read the code

Reply:
I'm not happy about it either, but what's the best way to make that explicit? This is important to let customers now the most popular types we can serializer. 
I also don't like adding a link to the code because we can change the code (add/remove lines) and the link will point somewhere that doesn't make sense.

Any idea?

Reply:
I think having an actual list of the types you have above is enough, but actually presented as a list.

Reply:
```suggestion
With data validation enabled, we natively support serializing the following data types to JSON:

Data type | Serialized type
------------------------------------------------- | ---------------------------------------------------------------------------------
**Pydantic models** | `dict`
**Python Dataclasses** | `dict`
**Enum** | Enum values
**Datetime** | Datetime ISO format string
**Decimal** | `int` if no exponent, or `float`
**Path** | `str`
**UUID** | `str`
**Set** | `list`
**Python primitives** _(dict, string, sequences, numbers, booleans)_ | [Python's default JSON serializable types](https://docs.python.org/3/library/json.html#encoders-and-decoders){target="_blank" rel="nofollow"}

???+ info "See [custom serializer section](#custom-serializer) for bringing your own."
    Otherwise, we will raise `SerializationError` for any unsupported types _e.g., SQLAlchemy models_.
```

editing: **1/** moved into a table, **2/** removed non-permalink to our encoder src, **3/** linked custom serializer, and **4/** added a note about `SerializationError`